editor/editor_properties_array_dict.cpp

```diff
@@ -424,8 +425,46 @@ void EditorPropertyArray::update_property() {
 			memdelete(container);
 			button_add_item = nullptr;
 			container = nullptr;
+			_update_property_bg();
+		}
+	}
+}
+
+void EditorPropertyArray::_update_property_bg() {
+	if (!is_inside_tree()) {
```

Comment:
This could potentially be a method of `EditorProperty`, so we don't have to duplicate everything. The only tricky part is the `sub_inspector`/`container` check, but that check can be extracted into a virtual method that only `EditorPropertyResource`, `EditorPropertyArray`, and `EditorPropertyDictionary` would implement.

Reply:
I succeeded in making it a method if `EditorProperty` but I had to make an adjustment. I made the sub_inspector from resource editor directly bottom_editor instead of being nested in a `VBoxContainer`. I'm not sure why it was the case but couldn't see any visual difference so I supposed it was okay to remove.